src/extensions/registries/__tests__/page-registry.test.ts

```diff
@@ -19,19 +19,18 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
+import { jest } from "@jest/globals";
```

Comment:
Is this needed or is this a better practice then what we do now?

Reply:
Yes, it's better practice to keep it explicitly + autocomplete in editors usually works better that way.